packaging/windows.cmake

```diff
@@ -0,0 +1,56 @@
+set(CPACK_GENERATOR "NSIS")
+find_package(cx_freeze 5.0 REQUIRED)
+
+configure_file(${CMAKE_CURRENT_LIST_DIR}/setup_win32.py.in setup.py @ONLY)
+add_custom_target(build_bundle)
+add_dependencies(packaging build_bundle)
+add_dependencies(build_bundle cura-binary-data)
```

Comment:
If I'm not wrong I found these lines somewhere in the old cura-build and as I didn't know why it was set like that I just kept it like that.
Missed to ask about it sorry. But yes, makes sense!